server/src/models.rs

```diff
@@ -0,0 +1,25 @@
+use diesel::prelude::*;
+
+#[derive(Queryable, Debug, Clone)]
+pub struct De {
+    pub key: String,
+    pub name: String,
+    pub arch_name: Option<String>,
+    pub type_: String,
+    pub type_common_name: String,
+    pub lat: f32,
+    pub lon: f32,
+    pub data: String,
+}
+
+#[derive(Queryable)]
+pub struct En {
+    pub key: String,
+    pub name: String,
+    pub arch_name: String,
+    pub type_: String,
+    pub type_common_name: String,
+    pub lat: f32,
+    pub lon: f32,
+    pub data: String,
+}
```

Comment:
I'm not sure, why do we need to separate that by language here, because `En` is never used in the code as far as I can see

Reply:
Yea, that is excessive. What name would you like? 
I think `RoomEntry` could be fitting…

Reply:
`RoomEntry` sounds good. Maybe also `DBRoomEntry` if we want to make clear it's from the database, but not necessarily.